scripts/mcptam_install.bash

```diff
@@ -81,35 +97,34 @@ install_dependencies()
 	install_toon
 	install_libcvd
 	install_gvars3
-	rm -rf build_deps
 	cd ..
+	rm -rf build_deps
+
 }
 
 build_mcptam()
 {
-	# setup ros env
-	source $ROS_BASH
-
-	# create catkin workspace
-	mkdir -p $HOME/catkin_ws/src
-	cd $HOME/catkin_ws/src
-	catkin_init_workspace
-	cd -
-
-	# setup catkin workspace
-	cd $HOME/catkin_ws/
-	catkin_make
-	source devel/setup.bash
-	cd -
-
-	# copy mcptam to catkin workspace
-	cd ..
-	cp -R mcptam $HOME/catkin_ws/src
-	cd $HOME/catkin_ws/
-	catkin_make
+
+	if [ "$CI" = true ]; then
+		# grab pull request version to be tested, build done by travis
+		cd ..
+	    cp -R mcptam $CATKIN_WS/src
+	    cd -
+	else
+		# clone mcptam to catkin workspace and build for user
+		cd $CATKIN_WS/src
+		git clone https://github.com/wavelab/mcptam
```

Comment:
I was trying to avoid duplicating the install script components for users and CI, hence this somewhat non-standard method.